diagnostic.md

```diff
@@ -73,20 +94,46 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl --request POST http://www.example.com/donuts \
+--header "Content-Type: application/json" \
+--data '{"name": "French Cruller", "price": "$0.99"}'
 ```
 
 Write an AJAX request to create a single donut on the server using JSON.
 
 ```js
-let createDonut = /* your answer here */;
+let createDonut = function() {
+  event.preventDefault();
+  $.ajax({
+    url: 'http://www.example.com/donuts',
+    type: 'POST',
+    data: {
+      'name': 'French Cruller',
+      'price': '$0.99'
+    }
+  }).done(logResponseBody).fail(logRequestError);
+};
 ```
 
 Now, rewrite your AJAX request to use FormData. Assume you have already targeted
 the correct HTML form before registering your AJAX function with the form.
 
 ```js
-let createDonut = /* your answer here */;
+let createDonut = function() {
+  event.preventDefault();
```

Comment:
You forgot to add `event` as an argument to the function; as written, `event` is undefined.